libs/postgres_ffi/src/lib.rs

```diff
@@ -163,6 +163,24 @@ pub fn page_set_lsn(pg: &mut [u8], lsn: Lsn) {
     pg[4..8].copy_from_slice(&(lsn.0 as u32).to_le_bytes());
 }
 
+pub fn fsm_logical_to_physical(addr: BlockNumber) -> BlockNumber {
```

Comment:
This function differs from the Postgres one. It lacks this piece of logic:

```
	/*
	 * If the page we were asked for wasn't at the bottom level, subtract the
	 * additional lower level pages we counted above.
	 */
	pages -= addr.level;
```
Is it safe, because caller will never provide non-bottom level page address?
Please, add a comment about it.

Reply:
Done